src/data/features.ts

```diff
@@ -151,12 +165,45 @@ export const tags = [
 		name: "Green",
 		color: "#77ac30"
 	},
-	{
-		name: "Azure",
-		color: "#4dbeee"
-	},
 	{
 		name: "Gray",
 		color: "#9ea3a1"
+	},
+	{
+		name: "Orange",
+		color: "#d95319"
+	}
+];
+
+export const taggedFiles: TaggedFileData[] = [
+	{
+		name: "Super Secret Plans - Do Not Publish.docx",
+		icon: "file",
+		tag: "Yellow"
+	},
+	{
+		name: "Presentation - Project Sledgehammer.pptx",
+		icon: "file",
+		tag: "Green"
+	},
+	{
+		name: "Files (MacOS)",
+		icon: "folder",
+		tag: "Gray"
+	},
+	{
+		name: "Microsoft Purchase Offer.pdf",
```

Comment:
I wouldn't use this as example/feature but that's just my 2cents.

Reply:
IIRC, Yair came up with this one, though we can use practically anything.